home.html

```diff
@@ -0,0 +1,140 @@
+
+        </nav>
+
+
+        <div class="conatiner1">
+            <div class="center">
+                <P class="paragraph">
+                    #Get Your 14 Days Free Trial
+                </P>
+                <h1 class="color">
+                    Manage All of your stuff using a TGB
+                </h1>
+                <p class="color text">
+                    Lorem ipsum dolor, sit amet consectetur adipisicing elit. Quo aliquam porro a fuga similique vero
+                    consectetur
+                </p>
+                <div class="option">
+                    <img class="smallimg" src="Images/i.png">
+                    <img class="smallimg" src="Images/i.png">
+                    <img class="smallimg" src="Images/i.png">
+                    <div class="color">4M+ user</div>
+                </div>
+
+                <button class="secbutton"> Learn More</button>
+
+            </div>
+            <img class="sideimage" src="Images/back.jpg">
+        </div>
+    </div>
+
+
+
+    <!-- here the Body Part start of webpage -->
+    <div class="main">
+
+        <div class="imagecontain">
+            <div class="leftdiv">
+                <h1 class="color1">EVENTS AND SHOWS</h1>
+                <p class="color1 jus">Lorem ipsum dolor sit amet consectetur adipisicing elit. Quos molestiae     omnis ipsum,
+                    illo reprehenderit natus asperiores ex! Illum itaque magni facilis ipsa non eaque minus quam! Totam
+                    corrupti beatae repudiandae id dolores ipsum officiis nostrum autem quos asperiores assumenda eum
+                    fuga provident at necessitatibus dolor est quo, incidunt amet aspernatur, consectetur quasi officia
+                    voluptatibus. Numquam voluptatum porro rem itaque, et possimus modi suscipit ea laboriosam id
+                    nesciunt. Nam omnis alias voluptate perferendis iste mollitia. Nihil provident ab maiores libero,
+                    ipsam nulla tempore est impedit sapiente assumenda nam eveniet unde. Dolorem eos dignissimos quaerat
+                    amet, debitis repellendus aliquam iure nemo unde.
+                </p>
+            </div>
+        </div>
+
+        <div class="body_div">
+            <div class="createflex">
+                <div class="card">
+                    <h1 class="color2">15</h1>
+                    <h1 class="color2">March</h1>
+                </div>
+                <div class="card2">
+                    <h3 class="color3">Where what is happen</h3>
+                    <p class="color3">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Et modi nihil cumque
+                        consectetur voluptatum harum explicabo at facilis similique libero, earum,
+                        <br />
+                        <button class="finalbut">Let's go</button>
+                    </p>
+                </div>
+            </div>
+
+            <div class="createflex">
```

Comment:
Please give proper naming. createflex is not a good name.

And, for all class and ids name use kebab case.